homeassistant/components/frontend/__init__.py

```diff
@@ -23,7 +23,7 @@
 from homeassistant.core import callback
 from homeassistant.loader import bind_hass
 
-REQUIREMENTS = ['home-assistant-frontend==20180130.0', 'user-agents==1.1.0']
+REQUIREMENTS = ['home-assistant-frontend==20180209.0', 'user-agents==1.1.0']
```

Comment:
Does this include chartsjs? If yes - it is possible to use an earlier version? This is a significant change - we should let it soak in dev

Reply:
It does not. I tagged it before I merged the chart js